packages/generator/ni_measurement_plugin_sdk_generator/client/_support.py

```diff
@@ -13,26 +13,27 @@
 from google.protobuf import descriptor_pool
 from google.protobuf.descriptor_pb2 import FieldDescriptorProto
 from google.protobuf.type_pb2 import Field
-from ni_measurement_plugin_sdk_service._internal.grpc_servicer import frame_metadata_dict
+from ni_measurement_plugin_sdk_service._internal.grpc_servicer import (
+    frame_metadata_dict,
+)
 from ni_measurement_plugin_sdk_service._internal.stubs.ni.measurementlink.measurement.v2 import (
     measurement_service_pb2 as v2_measurement_service_pb2,
     measurement_service_pb2_grpc as v2_measurement_service_pb2_grpc,
 )
 from ni_measurement_plugin_sdk_service.discovery import DiscoveryClient
 from ni_measurement_plugin_sdk_service.grpc.channelpool import GrpcChannelPool
 from ni_measurement_plugin_sdk_service.measurement.client_support import (
+    ParameterMetadata,
     create_file_descriptor,
     deserialize_parameters,
-    ParameterMetadata,
 )
 
-
 _V2_MEASUREMENT_SERVICE_INTERFACE = "ni.measurementlink.measurement.v2.MeasurementService"
 
 _INVALID_CHARS = "`~!@#$%^&*()-+={}[]\\|:;',<>.?/ \n"
 
 _XY_DATA_IMPORT = "from ni_measurement_plugin_sdk_service._internal.stubs.ni.protobuf.types.xydata_pb2 import DoubleXYData"
-_PATH_IMPORT = "from pathlib import Path"
+_PATH_IMPORT = "import pathlib"
```

Comment:
We hard-coded the import pathlib in #871 but continued iterating the `built_in_import_modules` dict until it got removed in #900. Therefore, this _PATH_IMPORT is not used/required anymore. Now, the `built_in_import_modules` is only used to check if it contains the "import pathlib" string literal.

Also, the `built_in_import_modules` currently only holds the "import pathlib" string literal. So, possibly in a separate PR can we replace the `built_in_import_modules` with a boolean which could be used to check if the measurement has a "Path" type configuration or output?

Reply:
I think I removed the `built_in_import_modules` checks but I will leave further cleanup for a separate PR, because it's not critical to do that before release.